tests/test_ophys/test_tools_roiextractors.py

```diff
@@ -1427,6 +1469,25 @@ def test_add_one_photon_series_roundtrip(self):
             expected_one_photon_series_shape = (self.num_frames, self.num_columns, self.num_rows)
             assert one_photon_series.shape == expected_one_photon_series_shape
 
+    def test_add_multiple_one_photon_series_with_same_imaging_plane(self):
+        """Test adding two OnePhotonSeries that use the same ImagingPlane."""
+        add_photon_series(
+            imaging=self.imaging_extractor,
+            nwbfile=self.nwbfile,
+            metadata=self.one_photon_series_metadata,
+            photon_series_type="OnePhotonSeries",
+        )
+        second_photon_series_metadata = deepcopy(self.one_photon_series_metadata)
+        second_photon_series_metadata["Ophys"]["OnePhotonSeries"][0]["name"] = "second_photon_series"
+        add_photon_series(
+            imaging=self.imaging_extractor,
+            nwbfile=self.nwbfile,
+            metadata=second_photon_series_metadata,
+            photon_series_type="OnePhotonSeries",
+        )
+
+        self.assertIn("second_photon_series", self.nwbfile.acquisition)
+
```

Comment:
Do you think it would be a good idea to check that we only have one imaging plane here written?

Something like 
```
assert len(self.nwbfile.imaging_planes) == 0
assert self.nwbfile.imaging_planes[0]["name"] == "the_default_name_or_the_name_we_chosen_in_this_test"
```

The second line is probably wrong but you get the idea.

Reply:
Yes, we should add a check for that! Thanks